pkg/asset/agent/image/cache.go

```diff
@@ -249,7 +249,8 @@ func (u *urlWithIntegrity) download(dataType string) (string, error) {
 
 // DownloadImageFile is a helper function that obtains an image file from a given URL,
 // puts it in the cache and returns the local file path.  If the file is compressed
-// by a known compressor, the file is uncompressed prior to being returned.
+// by a known compressor, the file is uncompressed prior to being returned.  The download
+// is retried 3 times in case of an error.
```

Comment:
Nit, it's only retried 2 times (but tried 3 times) at most. We should update the comment.

Reply:
Done!